### ISSUE TITLE
bug - Blank pages on loading book

### ISSUE THREAD

**Issue by DerekCaelin** [nvts8a/libros#29]
Upon loading any .txt file, Open Book displays pagination modal, and after process all pages are blank.]

## Steps To Reproduce
1. Be running v0.9.1 
2. Load [a book file](https://github.com/nvts8a/libros/files/11862489/bookchin-social-ecology-and-communalism.txt)
3. Say "yes" to pagination modeal
4. wait for pagination to finish
5. See issue

## Expected behavior
Load book with text

## Actual behavior
See mostly blank page, with a progress bar at the bottom and the correct number of pages. Can go forward and back but don't see any text.

![PXL_20230627_014458256](https://github.com/nvts8a/libros/assets/10799815/1bb1d208-2bcd-4431-87fd-568c53b4cfdc)

No pagination files appear on the sd card.

**Comment by nvts8a** [comment]
Calling out here as a reminder, [Issue #15](https://github.com/nvts8a/libros/issues/15) we do see issues with parsing sometimes with Windows file sources that hopefully we can solve soon. 

[Maybe a tool like this](https://toolslick.com/conversion/text/dos-to-unix) might help in the short term if you don't have the ability to edit files on anything but a Windows system

**Comment by DerekCaelin** [comment]
Hi @nvts8a thanks for thinking about this. Here's what happened. 

> Download and try a [short story](https://github.com/nvts8a/libros/tree/main/test/resources/short-stories) that I use to test, I use these all the time, seeing if the problem persist or doesn't would help with next steps

This is the resulting home screen:
![PXL_20230627_015451602 MP](https://github.com/nvts8a/libros/assets/10799815/d2c2e24f-57c1-444e-80d4-40fe3a3efe49)

I used [the Neil Gaiman short story](https://github.com/nvts8a/libros/blob/main/test/resources/short-stories/neil-gaiman-i-cthulhu.txt) in the directory you listed. The file displays as `{"payload": {'allShortcutsEnabled` on the home screen 

When I load the file, it asks to paginate, and then loads 25 pages of blank screen. 

> Download and try a the file you linked from GitHub as a source, rather than the one you have locally, see if the results change

I've done so: It asks to paginate, then loads 330 pages of blank screens.

> Take the entire contents of your SD card and zip it up and post it here, that should let me unzip it onto a card and see exactly what as has happened, there's a log file ./_OPENBOOK/openbook.log that might have some details as well

[Archive.tar.gz](https://github.com/nvts8a/libros/files/11876839/Archive.tar.gz)
Including the log: 
[openbook.log](https://github.com/nvts8a/libros/files/11876894/openbook.log)

I hope this is helpful. Maybe I should just start from scratch. What is the equivalent of doing a factory reset?

**Comment by DerekCaelin** [comment]
Thanks for this guidance!

I tested with the original firmware and v0.7.4 and encountered the same issue.

BUT I wiped everything from the SD card except [for the test file](https://github.com/nvts8a/libros/files/11862489/bookchin-social-ecology-and-communalism.txt) and I can now view the file.

![PXL_20230627_191704999](https://github.com/nvts8a/libros/assets/10799815/fe39b6ef-b8c3-4448-89c0-58860b725415)

I appreciate the support for early code and I hope filing a bug was helpful. I hope can contribute to the product in other ways? Closing this issue as resolved.
